Fragments/Filter/LHE_Gamma_Pt100to200_filter.dat

```diff
@@ -0,0 +1,9 @@
+LHE_Gamma_Pt100to200_filter = cms.EDFilter("LHE_Gamma_Pt100to200_filter",
```

Comment:
The specifiction in EDFilter("blabla") blabla should correspond to the official name of the filter, not your name, no ?

Reply:
fixed